.github/workflows/php.yml

```diff
@@ -0,0 +1,47 @@
+name: PHP Checks
+
+on: [ push, pull_request ]
+
+permissions:
+  contents: read
+
+jobs:
+
+  phpstan_pimcore10:
```

Comment:
Please rename this to something neutral

Reply:
Absolutely, copy-pasted this from another project